packages/ethereum/deployments/default/goerli/HoprChannels.json

```diff
@@ -1,7 +1,5 @@
 {
-  "address": "0xcBdD665ebEA87d3e028b1d92d463F40f1B273912",
-  "transactionHash": "0x6a25e5e4057ac7335ea17069309e3bdf61c3c83617c98e8ebeb864ca2bb1afce",
-  "blockNumber": 5376311,
```

Comment:
not sure how you have generated this file but we should neither drop `blockNumber` nor `transactionHash`